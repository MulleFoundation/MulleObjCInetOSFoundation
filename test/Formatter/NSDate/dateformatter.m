//
//  main.m
//  archiver-test
//
//  Created by Nat! on 19.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//


#import <MulleObjCOSFoundation/MulleObjCOSFoundation.h>
//#import "MulleStandaloneObjCFoundation.h"




int   main( int argc, const char * argv[])
{
   NSDateFormatter   *formatter;
   NSDate            *date;

   formatter = [[[NSDateFormatter alloc] initWithDateFormat:@"%Y-%m-%dT%H:%M:%SZ"
                                       allowNaturalLanguage:NO] autorelease];

   date = [formatter dateFromString:@"2013-02-24T20:09:15Z"];
   printf( "Date: %s\n", [[formatter stringFromDate:date] UTF8String]);

   return( 0);
}
