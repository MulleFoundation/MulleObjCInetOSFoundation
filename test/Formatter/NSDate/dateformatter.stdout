Date: 2013-02-24T20:09:15Z
